29-r_markdown_formats.Rmd

```diff
@@ -2,7 +2,14 @@
 
 **Learning objectives:**
 
-- Jon will try to prefill these ahead of each week.
+- **Control R Markdown output** with **output formats** and **options.**
+- Recognize the available R Markdown **document formats.**
```

Comment:
"recognize" always gives me pause. I feel like it's a weaker form of "list" or "identify", which ...are more like questions than LOs? Not sure what to suggest, though.

Reply:
I use it when you're supposed to read a thing and be able to maybe answer some multiple choice questions about it, but you aren't really expected to *know* the thing. I'm sure there's a better way to say it but I think this is good enough 🙃